scala/scala.bzl

```diff
@@ -298,6 +299,25 @@ def _write_launcher(ctx, jars):
         content=content)
 
 
+def _write_junit_test_launcher(ctx, jars, test_suite):
+    content = """#!/bin/bash
+{java} -cp {cp} -ea {archiveFlag} {prefixesFlag} {suffixesFlag} {printFlag} {jvm_flags} org.junit.runner.JUnitCore {test_suite_class} 
+"""
+    content = content.format(
+      java=ctx.file._java.short_path,
+      cp=":".join([j.short_path for j in jars]),
+      test_suite_class=test_suite.suite_class,
+      archiveFlag = test_suite.archiveFlag,
+      prefixesFlag = test_suite.prefixesFlag,
+      suffixesFlag = test_suite.suffixesFlag,
+      printFlag = test_suite.printFlag,
+      #allows setting xmx for example for tests which use a lot of memory
+      jvm_flags = " ".join(ctx.attr.jvm_flags)
+    )
+    ctx.file_action(
+        output=ctx.outputs.executable,
+        content=content)
```

Comment:
(Depending on PR submission order one of us will have to remember this) this will be the call to the unified _write_launcher in #171:

mandatoryJvmFlags = ["-ea", test_suit.archiveFlag, test_suite.prefixesFlag, test_suite.suffixesFlag, test_suite.printFlag]
 _write_launcher(
      ctx = ctx,
      rjars = jars,
      main_class = "org.junit.runner.JUnitCore",
      jvm_flags = mandatoryJvmFlags + ctx.attr.jvm_flags,
      args = test_suite.suite_class,
      run_before_binary = "",
      run_after_binary = "",
  )